test/bats/configs/vault-kv-secretproviderclass-jwt-auth.yaml

```diff
@@ -11,8 +11,9 @@ spec:
   parameters:
     roleName: "jwt-kv-role"
     vaultAuthMountPath: "jwt"
+    audience: "vault"
     objects: |
       - objectName: "secret-1"
         secretPath: "secret/data/kv1"
         secretKey: "bar1"
-        filePermission: 0600
+        filePermission: 0644
```

Comment:
Changed the file permissions in some of the tests since the OpenShift UBI image is running as a non-root user, but when the secrets store CSI driver writes files they're always owned by root: https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/858